docs/cloud/custom-domain.md

```diff
@@ -59,3 +59,7 @@ You'll still be able to access your original, automatically provisioned domain (
 ## Change the name of your Metabase (your DNS alias)
 
 Log in to your Metabase Store account and go to **Instances** > **DNS Alias**, where you can update your alias.
+
+## Custom SMTP server
+
+By default, Metabase Cloud manages an SMTP server for you. But if you want to change the address Metabase uses to send email, you can bring your own [custom SMTP server](../configuring-metabase/email.md#custom-smtp-server-on-metabase-cloud).
```

Comment:
Couple comments:
- it isn't super obvious that you can't change the from address because Metabase manages the SMTP server
- there are other reasons to use your own SMTP server besides changing the from address (but this is the main reason).

I think we could go with something like:

By default, Metabase Cloud manages an SMTP server for you. But you want to use your own SMTP server, you can bring your own.
-- and a few reasons someone would want to use their own SMTP server:
- customize from domain
- avoid passing through 3rd-party services
- own IP reputation, logs, monitoring
- own auditing